samples/matter/lock/README.rst

```diff
@@ -234,6 +234,10 @@ The PIN code is different depending on the build type:
 
 See `Testing door lock using Bluetooth LE with Nordic UART Service`_ for more information about how to test this feature.
 
+.. note::
+   This sample supports one Bluetooth LE connection at a time.
+   Matter commissioning, DFU, and NUS over Bluetooth LE have to be run separately.
```

Comment:
```suggestion
.. note::
   This sample supports one Bluetooth LE connection at a time.
   Matter commissioning, DFU, and NUS over Bluetooth LE must be run separately.
```
All good, but I would place this in line 223, with appropriate empty lines before and after.